src/glmfit.jl

```diff
@@ -439,6 +443,7 @@ const FIT_GLM_DOC = """
     for a list of built-in links).
 
     # Keyword Arguments
+    - `dropcollinear::Bool=false`: Controls whether or not lm accepts a model matrix which is less-than-full rank.
```

Comment:
Let's use the full description as for `?lm`:
```suggestion
    - `dropcollinear::Bool=false`: Controls whether or not `lm` accepts a model matrix which
      is less-than-full rank. If `true` (the default), only the first of each set of
      linearly-dependent columns is used. The coefficient for redundant linearly dependent columns is
      `0.0` and all associated statistics are set to `NaN`.
```